models/Element/Tag.php

```diff
@@ -319,4 +342,51 @@ public function correctPath()
             $this->idPath = '/';
         }
     }
+
+    /**
+     * @return mixed
+     */
+    public function getTranslations() {
+        return $this->translations ? unserialize($this->translations) : [];
+    }
+
+    /**
+     * @param mixed $translations
+     */
+    public function setTranslations($translations) {
+        $this->translations = $translations;
```

Comment:
The fact that the values are serialized internally should be transparent for the user of this class.

```suggestion
        $this->translations = serialize($translations);
```